lib/DDG/Spice/Zomato.pm

```diff
@@ -0,0 +1,52 @@
+package DDG::Spice::Zomato;
+# ABSTRACT: Write an abstract here
+# Start at https://duck.co/duckduckhack/spice_overview if you are new
+# to instant answer development
+
+use strict;
+use DDG::Spice;
+
+name "Restaurants Search";
+source "Zomato";
+icon_url "https://www.zomato.com/images/logo/zomato_favicon3.png";
+description "Suggest restaurants or places to eat in and around a locality";
+primary_example_queries "restaurants in gurgaon", "restaurants in bangalore";
+secondary_example_queries "restaurants nearby gurgaon","places to eat in gurgaon","food places in gurgaon";
+category "food";
+topics "food_and_drink","entertainment","social";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Zomato.pm";
+attribution github => ["Zomato", "Zomato"],
+            twitter => "Zomato",
+            facebook => "zomato",
+            web => ["https://www.zomato.com","Zomato"];
+
+
+spice is_cached => 0;
+spice to => 'http://dheerajavvari.zdev.net/duckduckgo_api.php?q=$1&lat=$2&lon=$3&lang_code=$4&rtl=$5&callback={{callback}}';
```

Comment:
There are few cases where same city name is there in different countries. For example, "Delhi", it is there in India, Canada and also in US. To parse more accurately in such situations, current location of the user will be very handy.